### DISCUSSION TITLE
Welcome to hactoberfest Discussions!

### DISCUSSION THREAD

**Discussion by atapas** [reactplay/hacktoberfest#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by buharisayed** [comment]
Hi Everyone, I'm Buhari Sayed a React Developer

**Reply by atapas** [reply]
Hello @buharisayed 

Do you want to join our Discord? All discussions moved there.

Link: https://discord.gg/UfYj4MvW9A
